episodes/17-live.md

```diff
@@ -107,7 +123,7 @@ Many error messages are obscure and not written with novices in mind. Continue t
 
 ## Compare and Contrast
 
-Watch this first participatory live coding demo video: [https://youtu.be/bXxBeNkKmJE][live-coding-bad]
+Watch this first participatory live demo video: [https://youtu.be/bXxBeNkKmJE][live-coding-bad]
```

Comment:
this is actually code, so I think here, live "coding" is actually more accurate and we should augment the exercise with reflection on what we can learn from this that applies to non-coding contexts